app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt

```diff
@@ -136,23 +136,12 @@ fun MessagePreview.uiLastMessageContent(): UILastMessageContent {
                                 else R.string.last_message_other_user_shared_video
                             )
                         )
-
-                    AssetType.ASSET ->
-                        UILastMessageContent.SenderWithMessage(
-                            userUIText, UIText.StringResource(
-                                if (isSelfMessage) R.string.last_message_self_user_shared_asset
-                                else R.string.last_message_other_user_shared_asset
-                            )
-                        )
-
-                    AssetType.FILE ->
-                        UILastMessageContent.SenderWithMessage(
-                            userUIText,
-                            UIText.StringResource(
-                                if (isSelfMessage) R.string.last_message_self_user_shared_file
-                                else R.string.last_message_other_user_shared_file
-                            )
+                    AssetType.GENERIC_ASSET -> UILastMessageContent.SenderWithMessage(
```

Comment:
not sure whre this have changed, maybe @gongracr can help

Reply:
I assume it will go away if you just pull from latest `develop` state 😅